src/test/java/io/jenkins/plugins/artifact_manager_s3/JCloudsBlobStoreTest.java

```diff
@@ -24,31 +24,33 @@
 
 package io.jenkins.plugins.artifact_manager_s3;
 
+import static org.hamcrest.Matchers.*;
```

Comment:
Urgh, we need to adopt a consistent import format guideline. I generally just put them all in one block, sorted, as it is simplest and least vulnerable to merge conflicts.

Reply:
I use the default Eclipse Java guidelines, with longer lines IIRC